Makefile

```diff
@@ -119,7 +119,7 @@ fmt: ## Run go fmt against code.
 .PHONY: vet
 vet: gowork ## Run go vet against code.
 	go vet ./...
```

Comment:
I'm wondering if running:

```
go vet ./... ./api/...
```

is equivalent to:

```
go vet ./...
go vet ./api/...
```

Reply:
no issue just normalized to the pattern other repos had where they had one folder per call.  but don't care one way or the other.

Reply:
was just simplifying ... either remove the redundant ./... from the double call or remove the call above it